Demo/Demo/FeatureViewControllers/PaymentButtonCustomizationViewController.swift

```diff
@@ -0,0 +1,213 @@
+                edgesPicker,
+                sizePicker
+            ]
+        )
+        stackView.axis = .vertical
+        stackView.alignment = .leading
+        stackView.spacing = 16.0
+        stackView.distribution = .fill
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UIEdgeInsets(
+            top: 16.0,
+            left: 16.0,
+            bottom: 16.0,
+            right: 16.0
+        )
+        return stackView
+    }()
+
+    // MARK: - Init
+
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        configure()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Initialization
+
+    private func configure() {
+        // Add subviews
+        scrollView.addSubview(stackView)
+        view.addSubview(scrollView)
+
+        view.backgroundColor = .white
+
+        // Add constraints
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate(constraints())
+
+        stackView.addArrangedSubview(paymentButton)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        reloadColorPicker()
+    }
+
+    // MARK: - Constraints
+
+    // In order to self size scroll view, constraint UIScrollView to all four corners
+    // of parent view constraint, then stackView to top bottom sides of parent scroll view
+    // content layout guide, centerX of scrollView and set width and height equal
+    // to frame layout guide. Ensure that stack view height constraint has a low priority.
+    private func constraints() -> [NSLayoutConstraint] {
+        let stackViewHeight = stackView.heightAnchor.constraint(
+            equalTo: scrollView.frameLayoutGuide.heightAnchor
+        )
+        stackViewHeight.priority = .defaultLow
+        return [
+            scrollView.topAnchor.constraint(equalTo: view.topAnchor),
+            scrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            stackViewHeight,
+            stackView.widthAnchor.constraint(
+                equalTo: scrollView.frameLayoutGuide.widthAnchor
+            ),
+            stackView.leadingAnchor.constraint(
+                equalTo: scrollView.contentLayoutGuide.leadingAnchor
+            ),
+            stackView.trailingAnchor.constraint(
+                equalTo: scrollView.contentLayoutGuide.trailingAnchor
+            ),
+            stackView.topAnchor.constraint(
+                equalTo: scrollView.contentLayoutGuide.topAnchor
+            ),
+            stackView.bottomAnchor.constraint(
+                equalTo: scrollView.contentLayoutGuide.bottomAnchor
+            )
```

Comment:
Nit: I think we can single line these and not hit our line length linter warnings